tests/test-cases-lib.sh

```diff
@@ -75,6 +107,61 @@ function TEST_road-traffic-predictor() {
 
 }
 
+function TEST_realtime-trade-monitor() {
+  echo "********** START TEST: realtime-trade-monitor **********"
+
+  local hzVersion=5.0-SNAPSHOT
+  local timeout=200
+  local pause=20
+  local demo_root=$1
+  local log_file=realtime-trade-monitor-log.txt
+
+  echo "---------- realtime-trade-monitor: preparing env - starting kafka service and creating \"trades\" topic"
+  cd $demo_root/tests/tools/qe-kafka-manager
+  timeout $timeout mvn exec:java -DkafkaTopic=trades&
+  local KAFKA_PID=$!
+  sleep $pause
+
+  echo "---------- realtime-trade-monitor: preparing env - starting trades producer"
+  cd $demo_root/realtime-trade-monitor
+  timeout $timeout java -jar trade-producer/target/trade-producer-${hzVersion}.jar 127.0.0.1:9092 5 | tee ${log_file}&
+  local PRODUCER_PID=$!
+  sleep $pause
+
+  echo "---------- realtime-trade-monitor: starting jet server"
+  timeout $timeout java -jar jet-server/target/jet-server-${hzVersion}.jar | tee ${log_file}&
+  local JET_PID=$!
+  sleep $pause
+
+  echo "---------- realtime-trade-monitor: starting jet jobs to ingest and aggregate trades"
+  java -jar trade-queries/target/trade-queries-${hzVersion}.jar load-symbols | tee ${log_file}
+  sleep $pause
+  java -jar trade-queries/target/trade-queries-${hzVersion}.jar ingest-trades 127.0.0.1:9092 | tee ${log_file}
+  sleep $pause
+  java -jar trade-queries/target/trade-queries-${hzVersion}.jar aggregate-query 127.0.0.1:9092 | tee ${log_file}
+  sleep $pause
+
+  echo "********** Launch data-observer to make sure landingMap is being updated by job **********"
+  cd $demo_root/tests/tools/qe-data-observer/target || exit
```

Comment:
Yes, I added this as an experiment. Let's discuss what way is preferable, I think we have to log error and exit, but this will make the code a bit longer.

Reply:
Discussed. Changing to exit 1